### DISCUSSION TITLE
[projects beta] - import/export to csv file or clipboard

### DISCUSSION THREAD

**Discussion by cigumo** [community/community#6217] [Projects and Issues] [answered]
It would be very useful to copy/paste from a spreadsheet into projects beta and viceversa. 
It could be either going through a csv file or directly if possible. 

When managing projects I normally estimate the cost in a spreadsheet by detailing each feature or task. 
Copying and pasting this into github projects would save me the time to retype everything from scratch, which is clearly a game breaker for large projects.  

Thanks!

**Comment by malkoG** [comment]
+1 :)

**Comment by ghost** [comment]
+ 1 :)

**Comment by pascalpvk** [comment]
And Discussion / Description / Tags / Progress should be included in those CSV export

**Comment by rileybroughten** [comment, marked as answer]
You can now export your project view to a CSV file! Check it out on the [changelog](https://github.blog/changelog/2023-06-15-github-issues-projects-june-15th-update/#%F0%9F%8F%97%EF%B8%8F-export-project-view-as-a-csv-file) announcement and let us know what you think 🚀

**Reply by jrslhelcom** [reply]
its not so anylonger. also useless because the "meant" of the item which is the description is not exportable. again: useless.
